### PR TITLE
Consistent Region support for Kafka Consumer

### PR DESCRIPTION
**Pull request by Alex-Cook4**
See wiki for consistent region design: https://github.com/IBMStreams/streamsx.messaging/wiki/Consistent-Region-Support-for-Messaging-Toolkit

### CODE REVIEW

**Comment by ddebrunner** [comment]
Is the design going to be updated to describe the behaviour of KafkaProducer in a consistent region?

File: com.ibm.streamsx.messaging/.classpath [hunk 1]
```diff
@@ -18,8 +16,10 @@
 	<classpathentry kind="lib" path="opt/downloaded/activemq-client-5.11-SNAPSHOT.jar"/>
 	<classpathentry kind="lib" path="opt/downloaded/geronimo-j2ee-management_1.1_spec-1.0.1.jar"/>
 	<classpathentry kind="lib" path="opt/downloaded/geronimo-jms_1.1_spec-1.1.1.jar"/>
-	<classpathentry kind="lib" path="opt/downloaded/slf4j-api-1.7.5.jar"/>
 	<classpathentry kind="con" path="com.ibm.streams.java/com.ibm.streams.operator"/>
-	<classpathentry kind="lib" path="opt/downloaded/org.eclipse.paho.client.mqttv3-1.0.1-SNAPSHOT.jar"/>
+	<classpathentry kind="lib" path="opt/downloaded/hawtbuf-1.11.jar"/>
+	<classpathentry kind="lib" path="opt/downloaded/kafka_2.8.0-0.8.1.1.jar"/>
```
Comment (dlaboss) [inline com.ibm.streamsx.messaging/.classpath]: Kafka2.10-0.8.2.1 seems to be the current version.  Shouldn't we upgraded to it?

Comment (Alex-Cook4) [inline com.ibm.streamsx.messaging/.classpath]: I will take a look at that and follow up. Thanks.

**Comment by Alex-Cook4** [comment]
@ddebrunner I am not familiar with the Kafka Producer consistent region design, but I will look and try to update soon. This commit is only for KafkaConsumer.

**Pull request merged**